### ISSUE TITLE
How to generate GIFs with duplicate frames?

### ISSUE THREAD

**Issue by palxex** [python-pillow/Pillow#7809]
### What did you do?
I just want to create GIFs like https://github.com/radarhere/Pillow/blob/main/Tests/images/duplicate_frame.gif, which have two( or more) identical frames.

### What did you expect to happen?
I thought save with optimize=False it will disable the deduplication optimization. But it do not work( combined, see below ). Also other workarounds mentioned in https://github.com/python-pillow/Pillow/issues/2314, either manually specify duration list, or with different disposal options.

### What actually happened?
All the sequential identical frame was combined, which was exactly what I'm trying to prevent.

### What are your OS, Python and Pillow versions?

* OS:  Windows 11/WSL2 Debian sid
* Python: 3.12.2/3.11.8
* Pillow: 10.2 

Could you please give me a code snippet, generate the test GIF mentioned above exactly with pillow? Or its simply not possible?

**Comment by radarhere** [comment]
Removing duplicate consecutive frames is intended to be a feature, reducing file size without affecting the visual output.

It would be good to understand why you feel the need for something that, unless I'm misunderstanding you, no human looking at the image will be able to discern?

The only way I can think of so far would be by modifying one of our private methods at runtime - which is a fragile solution, as being not public, it might change in the future without warning.

**Comment by palxex** [comment]
Let me explain; I'm using GIF as middle format for a set of tools, which import/export 8bit asset for an old game. I need to ensure that when an animation be exported and then reimported, it's exactly same as before - not just visual, but also bitwise. 

Its surely an corner case; but as an example, this optimization also prevent pillow from being able to produce any kind of GIF that user may desire, even the test images we needs here. 

Is it possible adding an option to skip this optimization in future? Thanks in advance.

**Comment by palxex** [comment]
Apologies for my pool expression; my intention is not exporting/importing the same file, but rather to ensure the original asset A( to be export to GIF) and the asset B( reimported from the exported GIF of A ) are exactly same, which constitutes a specific path within the entire workflow. In other paths, I still need to utilize many PIL functions. 

I raised this issue in the hope that the maintainers would consider adding a shortcut for this particular use case (and also be able to generate GIFs required in testing). If this cannot be accommodated due to engineering considerations (which I completely understand), no worries, I have found another solution for my problem: changing the intermediary format from GIF to a sequence of PNGs.
